src/dispatch/static/dispatch/src/router/index.js

```diff
@@ -134,12 +134,10 @@ function loginBasic(to, from, next) {
   let token = localStorage.getItem("token")
   if (token) {
     store.dispatch("account/loginWithToken", token)
-    return
```

Comment:
In case token exists and user is authorized, I think he should continue navigating where he wanted. Eliminating the return statement will cause the user to be redirected to "login" regardless of the token existence.

Reply:
That makes sense, although I'd rather we explicitly send them to their original destination. I was also getting a redirect routing loop when I was navigating to `/login` initially (and it kept sending me back to `/login` since that was my original destination (and I already had a token). 

Let me try to make it a bit more explicit.

Reply:
Yep, the loop comes from calling multiple times "next()" at L140 and L142